lab_6_pipeline/pipeline.py

```diff
@@ -1,187 +0,0 @@
-"""
```

Comment:
why did you need to delete the stubs for the next lab? 
if there is no necessity, please bring it back